netci.groovy

```diff
@@ -353,6 +353,7 @@ def targetGroupOsMapInnerloop = ['netcoreapp': ['Windows_NT', 'Ubuntu14.04', 'Ub
     ['netcoreapp'].each { targetGroup ->
         ['Debug', 'Release'].each { configurationGroup ->
```

Comment:
Can we scope this to just debug or release by default? We want to try and keep our CI leg count minimum so we use a sparse matrix. 

Also I'm not sure if we can but ideally we would put these in https://github.com/dotnet/corefx/blob/master/buildpipeline/pipelinejobs.groovy.

Reply:
@weshaggard I added to pipelinejobs.groovy already.

This is blocking official builds, so I 'll drop one leg in a separate PR

Reply:
>This is blocking official builds

Yes please unblock the builds.

>@weshaggard I added to pipelinejobs.groovy already.

Perhaps I missed it but it would seem as though you added it to netci.groovy and not pipelinejobs.groovy. Can you point me at that change?

Reply:
@weshaggard #27656

Reply:
I know it is confusing but pipelines.json isn't the same as pipelines.groovy. pipelines.json is for our official builds and pipelines.groovy is our newer jenkins CI scripts. netci.groovy is our older CI scripts we are trying to phase out.